octopusdeploy/schema_deployment_action.go

```diff
@@ -244,6 +244,12 @@ func getActionSchema() (*schema.Schema, *schema.Resource) {
 				Type:             schema.TypeMap,
 				ValidateDiagFunc: warnIfIncludesRunOnServer(),
 			},
+			"sort_order": {
```

Comment:
Also, I understand zero is ok for devs, but I wonder if it would be nice for this to start at `1`?

Reply:
Honestly as long as its sequential I dont really care. If someone puts 0 there's currently no way to differentiate that with not provided, since integer it defaults to 0.

Reply:
you can add a validator to this field

Reply:
We support 0 on other fields so for the sake of consistency im going to leave it as-is for now